kore/test/Test/Kore/Step/Step.hs

```diff
@@ -224,20 +245,23 @@ test_applyRewriteRule_ =
     [ testCase "apply identity axiom" $ do
         let expect = Right [ OrPattern.fromPatterns [initial] ]
             initial = pure (mkElemVar Mock.x)
-        actual <- applyRewriteRule_ initial axiomId
+        actual <- applyRewriteRule_ applyRewriteRulesParallel initial axiomId
```

Comment:
Why did we make this change? We don't seem to call `applyRewriteRule_` with any argument other than `applyRewriteRulesParallel`.

Reply:
It's used with `applyRewriteRulesSequence` at line 347, but I will change it to make it look cleaner